src/renderer/components/player-settings/player-settings.js

```diff
@@ -195,7 +195,15 @@ export default defineComponent({
 
     screenshotFilenamePattern: function() {
       return this.$store.getters.getScreenshotFilenamePattern
-    }
+    },
+
+    commentAutoLoadConditionCurrentValue: function () {
+      return this.$store.getters.getCommentAutoLoadCondition
+    },
+
+    commentAutoLoadConditionEnabled: function () {
```

Comment:
Using string is intentional even though UI is a toggle
I am not excluding the possibility to introduce other values later
No DB change needed when that's the case